libs/features/placeholders.js

```diff
@@ -16,8 +16,10 @@ const fetchPlaceholders = (config, sheet) => {
       const json = resp.ok ? await resp.json() : { data: [] };
       if (json.data.length === 0) { resolve({}); return; }
       const placeholders = {};
+      if (!config.analyticLocalization) config.analyticLocalization = {};
```

Comment:
What is this property? Who approved this property going into the config object?

I fear we are making the config object a mess for no reason... and it's not just this one, but it's others as well. We should be shipping "really good defaults" and less optional stuff people must be aware of when they spin up a new project.

We are already littering our DOM with stuff almost no one cares about like `nopzn` and I fear this is more of that type of stuff.

@chrischrischris @rgclayton @narcis-radu

Reply:
I don't think "analytics localization" is anything placeholders should need to be aware of. This makes a tight coupling here and I think this is a bad idea. Most features should be portable and free of these types of dependencies.

Reply:
`analyticsLocalization` is just the same config property the martech metadata block is using.  So it is adding to the object that the block has already created.  Line 19 is there to prevent an error in case placeholders was used but the martech metadata block was not.